src/rgw/rgw_op.cc

```diff
@@ -3433,9 +3433,6 @@ void RGWCreateBucket::execute(optional_yield y)
    * recover from a partial create by retrying it. */
   ldpp_dout(this, 20) << "rgw_create_bucket returned ret=" << op_ret << " bucket=" << s->bucket.get() << dendl;
 
-  if (op_ret)
-    return;
-
```

Comment:
discussed in the bug scrub - we want to return here on any fatal errors, but continue on if op_ret is either EEXIST or ERR_BUCKET_EXISTS